src/locale/locales/de/messages.po

```diff
@@ -4493,15 +4493,15 @@ msgstr "Du hast noch keine Konten blockiert. Um ein Konto zu blockieren, gehe au
 
 #: src/view/screens/AppPasswords.tsx:87
 msgid "You have not created any app passwords yet. You can create one by pressing the button below."
-msgstr "Du hast noch keine App-Passwörter erstellt. Du kannst eines erstellen, indem du auf die Schaltfläche unten klickst."
+msgstr "Du hast noch keine App-Passwörter erstellt. du kannst eines erstellen, indem du auf die Schaltfläche unten klickst."
```

Comment:
Also here, is it OK to start the sentence with lowercase?

Reply:
i just used find & replace to change it and forgot to ignore sentences that start with Du
how embarrasing 🫢

Reply:
No worries :)